### PR TITLE
Fix script 404s

### PR DESCRIPTION
**Pull request by twiss**
Forgot to remove them in #246

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,9 +28,6 @@
 	<script type="text/javascript" src="scripts/cloud/dropbox.js" defer></script>
 
 	<!-- Lib -->
-	<script type="text/javascript" src="scripts/lib/kamino.js" defer></script>
-	<script type="text/javascript" src="scripts/lib/uuid.core.js" defer></script>
-	<script type="text/javascript" src="scripts/lib/message_channel.js" defer></script>
```
Comment (joshua-s) [inline index.html]: Isn't message_channel.js used?

**Comment by ferndot** [comment]
Ready for merge once it is rebased!

**Comment by twiss** [comment]
Rebased.

**Comment by twiss** [comment]
Thanks!

**Pull request merged**